### PR TITLE
Add filter icons

### PR DESCRIPTION
**Pull request by ricardoom**
NOTE: Probably some breaking changes here, updated the logout icon which might break some stuff
removed fill attributes that might do it too.
Otherwise: 
added missing pencil icon
updated the logout icon

### CODE REVIEW

**Comment by ricardoom** [comment]
its weird that @ktp242 cant be added as a reviewer on this...
weird.

**Comment by ktp242** [comment]
Oh you found me block you sending me PR!!!

lol just kidding. Let's solve it tomorrow.

**Comment by ricardoom** [comment]
no more @rhernand3z to review our PRs...

sad.

File: src/components/FilterCancelIcon.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  height: PropTypes.string,
+  title: PropTypes.string,
+  width: PropTypes.string,
+  viewBox: PropTypes.string,
+};
+
+FilterCancelIcon.defaultProps = {
+  ariaHidden: true,
+  className: 'filter-cancel-icon',
+  height: '71.854',
```
Comment (EdwinGuzman) [inline src/components/FilterCancelIcon.jsx]: How'd you get this number?

Comment (ricardoom) [reply src/components/FilterCancelIcon.jsx]: on closer inspection, not sure. going back to source files and updating so this PR can get finished

Comment (EdwinGuzman) [reply src/components/FilterCancelIcon.jsx]: 👍

**Comment by ricardoom** [comment]
@EdwinGuzman please take another look. I think the new filter cancel sizing might be breaker for the dev-discovery filtering. let me know.
once you approve, i'll make the necessary updates to readme & changelog on my release version.

File: package.json [hunk 2]
```diff
@@ -9,6 +9,25 @@
     "styleguide": "node_modules/.bin/styleguidist server",
     "styleguide:build": "node_modules/.bin/styleguidist build"
   },
+  "author": "NYPL Digital Experience",
+"contributors": [
```
Comment (EdwinGuzman) [inline package.json]: Just indent this properly.

**Comment by EdwinGuzman** [comment]
It might be a breaker for discovery but that just means we'd have to update the css for the app, which I think is okay. Besides, some apps are WAY behind on the version for this dependency and those may have more and larger issues to fix.

**Comment by ricardoom** [comment]
updated.

File: package.json (hunk 2 above)
Comment (ricardoom) [reply package.json]: done

**Pull request merged**